agents/agents/guard/guard.go

```diff
@@ -232,6 +414,37 @@ func (g Guard) Start(parentCtx context.Context) error {
 		return err
 	})
 
+	group.Go(func() error {
```

Comment:
group->g, let's keep this consistent

Reply:
Receiver name here is already `g`. Going to keep it as `group`, unless you think we should either a) change the receiver, b) change from `group` to something else (non `g`)